RELEASE/scripts/autoscend/auto_king.ash

```diff
@@ -3,112 +3,77 @@ import <autoscend.ash>
 {
 	restoreAllSettings();
-	if(inAftercore() && (get_property("auto_snapshot") == ""))
+	if(!inAftercore())
 	{
-		auto_log_info("Yay! The King is saved. I suppose you should do stuff.");
-		if(!get_property("auto_kingLiberation").to_boolean())
-		{
-			set_property("auto_snapshot", "aborted");
-			return;
-		}
-		visit_url("storage.php?action=takemeat&pwd&amt=" + my_storage_meat());
-		visit_url("storage.php?pwd&");
+		auto_log_info("can't run king liberated script. I am not actually in aftercore.");
+		return;
+	}
 
-		if(my_familiar() != $familiar[Black Cat])
-		{
-			set_property("auto_100familiar", $familiar[none]);
-		}
+	auto_log_info("Yay! The King is saved. I suppose you should do stuff.");
 
-		// Some items don\'t get pulled, notably, Stench Wad but some others as well (fat loot token, holiday fun, box of sunshine).
-		// This might fix it...
-		cli_execute("refresh all");
+	if(my_familiar() != $familiar[Black Cat])
+	{
+		set_property("auto_100familiar", "");
+	}
 
-		if(have_display())
+	if(have_display())
+	{
+		boolean[item] toDisplay = $items[Instant Karma, Thwaitgold Ant Statuette, Thwaitgold Bee Statuette, Thwaitgold Bookworm Statuette, Thwaitgold Butterfly Statuette, Thwaitgold Caterpillar Statuette, Thwaitgold Cockroach Statuette, Thwaitgold Dragonfly Statuette, Thwaitgold Firefly Statuette, Thwaitgold Goliath Beetle Statuette, Thwaitgold Grasshopper Statuette, Thwaitgold Maggot Statuette, Thwaitgold Moth Statuette, Thwaitgold Nit Statuette, Thwaitgold Praying Mantis Statuette, Thwaitgold Scarab Beetle Statuette, Thwaitgold Scorpion Statuette, Thwaitgold Spider Statuette, Thwaitgold Stag Beetle Statuette, Thwaitgold Termite Statuette, Thwaitgold Wheel Bug Statuette, Thwaitgold Woollybear Statuette];
+		foreach it in toDisplay
 		{
-			boolean[item] toDisplay = $items[Instant Karma, Thwaitgold Ant Statuette, Thwaitgold Bee Statuette, Thwaitgold Bookworm Statuette, Thwaitgold Butterfly Statuette, Thwaitgold Caterpillar Statuette, Thwaitgold Cockroach Statuette, Thwaitgold Dragonfly Statuette, Thwaitgold Firefly Statuette, Thwaitgold Goliath Beetle Statuette, Thwaitgold Grasshopper Statuette, Thwaitgold Maggot Statuette, Thwaitgold Moth Statuette, Thwaitgold Nit Statuette, Thwaitgold Praying Mantis Statuette, Thwaitgold Scarab Beetle Statuette, Thwaitgold Scorpion Statuette, Thwaitgold Spider Statuette, Thwaitgold Stag Beetle Statuette, Thwaitgold Termite Statuette, Thwaitgold Wheel Bug Statuette, Thwaitgold Woollybear Statuette];
-			foreach it in toDisplay
+			if(item_amount(it) > 0)
 			{
-				if(item_amount(it) > 0)
-				{
-					auto_log_info("Displaying " + item_amount(it) + " " + it, "green");
-				}
-				put_display(item_amount(it), it);
+				auto_log_info("Displaying " + item_amount(it) + " " + it, "green");
 			}
+			put_display(item_amount(it), it);
 		}
+	}
+	
+	if(get_property("lastEmptiedStorage").to_int() != my_ascensions())
+	{
+		cli_execute("pull all");
+	}
+	cli_execute("breakfast");
```

Comment:
Before calling breakfast, could you also put in some `visit_url` calls to talk to the old man at the sea? That way mafia will grab sea jelly as part of breakfast if you have a space jellyfish. I always find myself doing this manually, so that'd be nice.

Could make the `visit_url` calls only happen if you have a space jellyfish, or they could just happen regardless, whichever you think is better.

Reply:
change made